### ISSUE TITLE
[ENH] - Limit Search Address results to USA

### ISSUE THREAD

**Issue by pjdohertygis** [pjdohertygis/SARCOP#250]
### Which app?

Intel Manager 2.0 (Experience Builder)

### Describe requirements

Ideally the Intel Manager only searches for Addresses in the United States, this will speed up the process of finding addresses and places.

### Alternatives

Keep as is..

### Other Relevant Info

See this issue and fix https://github.com/pjdohertygis/DeploymentOrderTool/issues/4

### Priority impact

p0 - not time sensitive

### Impact

This will speed up the process of finding addresses and places and reduce confusion for the end user.

**Comment by pjdohertygis** [comment]
So far, positive feedback from UAT "The Tinker Air Force Base was an easy search".

**Comment by afackler** [comment]
Looks like the best way to achieve this is to make a geocoder based on the Esri World Geocoder for USA addresses only, then plug it in as the "Source" for the search widget. Otherwise we can't filter the World Geocoder by just US addresses
![image](https://github.com/pjdohertygis/SARCOP/assets/55416740/2e324abd-4601-44d6-a675-92cd6ab35210)

**Comment by pjdohertygis** [comment]
Adam - check the documentation, there should be an option for just limiting to “USA” in the country list vs a separate geocoder. Regards,Paul On Jul 5, 2024, at 13:46, Adam Fackler ***@***.***> wrote:﻿
Looks like the best way to achieve this is to make a geocoder based on the Esri World Geocoder for USA addresses only, then plug it in as the "Source" for the search widget. Otherwise we can't filter the World Geocoder by just US addresses
image.png (view on web)

**Comment by afackler** [comment]
After creating a view of the Esri World Geocoder, I was able to get it plugged into the experience and limit the search to just the US. The locator can be found here https://arcgis.com/home/item.html?id=0b164f8f4ae040429881c66557c3853a#overview. 
![image](https://github.com/pjdohertygis/SARCOP/assets/55416740/dbfbac75-b792-44ca-bb9c-8dd98a933208)

Once the geocoder was created, I removed the World Geocoder and attempted to put the new USA Addresses geocoder in, but kept receiving the "Invalid URL" error. After diging around, I found the geocoder had to be added to the Utility services, then the geocoder had to be added from the "Added" tab and not the Organization tab when adding a new source. 
![image](https://github.com/pjdohertygis/SARCOP/assets/55416740/37104c42-b3cc-48b3-8b32-b4f0adacf524)

When testing, it only shows the USA Addresses! Project saved and issue closed.

Before:
![image](https://github.com/pjdohertygis/SARCOP/assets/55416740/2983cde5-d410-4ba5-b0ba-21fe582e7f66)

After:
![image](https://github.com/pjdohertygis/SARCOP/assets/55416740/af37259e-a5d1-4713-a7ae-9d11341521b7)

**Comment by pjdohertygis** [comment]
Just be sure to test sharing of the geocoder (everyone?), performance, etc On Jul 11, 2024, at 09:12, Adam Fackler ***@***.***> wrote:﻿
After creating a view of the Esri World Geocoder, I was able to get it plugged into the experience and limit the search to just the US. The locator can be found here https://arcgis.com/home/item.html?id=0b164f8f4ae040429881c66557c3853a#overview.
image.png (view on web)
Once the geocoder was created, I removed the World Geocoder and attempted to put the new USA Addresses geocoder in, but kept receiving the "Invalid URL" error. After diging around, I found the geocoder had to be added to the Utility services, then the geocoder had to be added from the "Added" tab and not the Organization tab when adding a new source.
image.png (view on web)
When testing, it only shows the USA Addresses! Project saved and issue closed.
Before:
image.png (view on web)
After:
image.png (view on web)
